subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -74,6 +74,9 @@
 #define RADIO_TIFS    150 /* BT Spec. defined */
 /* Inter Event Space */
 #define RADIO_TIES_US 625 /* Implementation defined */
+/* Range Delay */
+#define RADIO_RANGE_DISTANCE 1000 /* meters */
+#define RADIO_RANGE_DELAY_US (2 * RADIO_RANGE_DISTANCE * 4 / 1000)
```

Comment:
Maybe write a comment on this expression so it seems less magically? Like that "4 / 1000" is an approximation of the propagation time in us of the signal to travel 1m.